README.md

```diff
@@ -36,32 +34,32 @@ This test suite uses the [official JSON-schema test suite](https://github.com/js
-|[`revalidator`](https://github.com/flatiron/revalidator)|[277](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/revalidator.md)|
+|[`jsonschema`](https://github.com/tdegrunt/jsonschema#readme)|[76](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/jsonschema.md)|
+|[`is-my-json-valid`](https://github.com/mafintosh/is-my-json-valid#readme)|[100](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/is-my-json-valid.md)|
+|[`djv`](https://github.com/korzio/djv#readme)|[100](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/djv.md)|
+|[`json-schema-library`](https://github.com/sagold/json-schema-library)|[126](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/json-schema-library.md)|
+|[`tv4`](https://github.com/geraintluff/tv4)|[129](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/tv4.md)|
+|[`jsen`](https://github.com/bugventure/jsen)|[130](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/jsen.md)|
+|[`jassi`](https://github.com/iclanzan/jassi)|[146](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/jassi.md)|
+|[`z-schema`](https://github.com/zaggino/z-schema)|[157](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/z-schema.md)|
+|[`jjv`](https://github.com/acornejo/jjv)|[160](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/jjv.md)|
+|[`themis`](https://github.com/playlyfe/themis)|[177](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/themis.md)|
+|[`request-validator`](https://github.com/bugventure/request-validator)|[181](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/request-validator.md)|
+|[`json-schema-validator-generator`](https://github.com/danwang/json-schema-validator-generator)|[190](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/json-schema-validator-generator.md)|
+|[`JSV`](http://github.com/garycourt/JSV)|[206](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/JSV.md)|
+|[`json-model`](https://github.com/geraintluff/json-model)|[206](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/json-model.md)|
+|[`jsck`](https://github.com/pandastrike/jsck#readme)|[214](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/jsck.md)|
+|[`skeemas`](https://github.com/Prestaul/skeemas#readme)|[221](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/skeemas.md)|
+|[`schemasaurus`](https://github.com/AlexeyGrishin/schemasaurus)|[222](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/schemasaurus.md)|
+|[`json-gate`](https://github.com/oferei/json-gate#readme)|[312](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/json-gate.md)|
+|[`revalidator`](https://github.com/flatiron/revalidator)|[355](https://github.com/ebdrup/json-schema-benchmark/blob/master/reports/revalidator.md)|
```

Comment:
This is a very strange result. Baseline `() => true` returns 358 failing, `revalidator` returns 355 failing.

Reply:
Well, same on master: `() => true` -- 287 failures, `revalidator` -- 277 failures.
Not this PR fault.

Reply:
I think this list can be pruned a bit - no much point keeping the bottom of the list. Also I am surprised djv fails so many tests - it was relatively compliant. I think it might be some API issue we could fix.